src/containers/App/App.jsx

```diff
@@ -45,6 +49,19 @@ const App = (props) => {
     }
   })
 
+  useEffect(() => {
+    const staticNetworkName = getNetworkName()
```

Comment:
Move the initialization getNetworkName outside of useEffect.  This will remove a guaranteed re-render on non custom network pages.